internal/pkg/cli/svc_logs.go

```diff
@@ -209,6 +209,7 @@ func (o *svcLogsOpts) Execute() error {
 			return err
 		}
 		o.taskIDs = []string{taskID}
+		fmt.Printf("previously stopped task: %s\n", taskID)
```

Comment:
delete?

Reply:
Ohh so we discussed in the meeting that we want to show taskId too as a part of `--previous` functionality. That is why I am displaying taskId before we show logs for the that stopped task.

Reply:
Ah, gotcha. I was thrown off by the `fmt.Printf`...should we use a `log.` instead?